library/boehm/build.rs

```diff
@@ -5,8 +5,8 @@ use std::process::Command;
 
 const BOEHM_REPO: &str = "https://github.com/softdevteam/bdwgc.git";
 const BOEHM_ATOMICS_REPO: &str = "https://github.com/ivmai/libatomic_ops.git";
-const BOEHM_DIR: &str = "bdwgc";
-const BUILD_DIR: &str = "lib";
+const BOEHM_DEFAULT_SRC_DIR: &str = "bdwgc";
```

Comment:
Should we consistently call this `BDWGC` (including the env var etc)?

Reply:
I think so really yeah. I'll push an update.

Reply:
Since this PR is really just a bunch of miscellaneous fixes, I've decided to go ahead and fix this across the entire codebase: 6cfff5627161f501f150bc9aeb0f66859297a613